SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,55 @@
+header:
```

Comment:
`documentation` link might be nice as well, like here: https://github.com/kubernetes-sigs/kueue/pull/1469/files#diff-000b1100271690c8bb5643a7f806f131a81fdb22cf27af740f5d645e985e9811R27

Reply:
https://karpenter.sh/docs